### PR TITLE
Update harmony.markdown

### LINKED ISSUE #21444: Add instructions for turning on XMPP in Harmony Hub
### Feedback

It took me a while to figure out that I need to turn on XMPP developer option in Harmony Hub to get this working. It'd be good to add that into the docs.

### URL

https://www.home-assistant.io/integrations/harmony/

### Version

2022.2.0

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by cliffjao**
Add note to inform people that turning on XMPP may be needed.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: #21444

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @cliffjao, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21484).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/harmony.markdown [hunk 1]
```diff
@@ -32,6 +32,8 @@ Supported units:
 
 {% include integrations/config_flow.md %}
 
+**Note:** Depending on the firmware, you may need to enable XMPP for this integration to work.
```
Comment (bdraco) [inline source/_integrations/harmony.markdown]: Maybe a note on how to do it?

From your Harmony app, go to: Menu > Harmony Setup > Add/Edit Devices & Activities > Remote & Hub > Enable XMPP

Comment (cliffjao) [reply source/_integrations/harmony.markdown]: Updated

**Review by bdraco — APPROVED** [review]
LGTM 👍 

Thanks @cliffjao

**Review by klaasnicolaas — APPROVED** [review]
✅ LGTM

**Comment by bdraco** [comment]
I should have renamed that one. 😨

**Pull request merged**